### PR TITLE
Wilson/paginated past campaigns

### PR DESCRIPTION
**Pull request by wilsonj806**
- Split off campaign fetching in GAM page so that it's making two requests, one for active campaigns and one for inactive campaigns.
- Update CampaignList item to use updated campaigns API and so it doesn't have to make 2 extra requests
- Small fix to ModalBuyMealPool styling

[Notion Page for screen images](https://www.notion.so/e0fc1a7243594292b27bea6c3094e27b)

Changes to data fetching are based on changes in this [pull request](https://github.com/sendchinatownlove/ruby/pull/304)

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -206,6 +206,11 @@
     },
     "backButton": "Back to Merchants",
     "giftButton": "Gift Meal",
+    "viewMoreButton": "View More",
```
Comment (jacoblb64) [inline src/locales/en/translation.json:209]: do we need these for `cn` translations too?

Comment (wilsonj806) [reply src/locales/en/translation.json:209]: I'm actually not sure and included it to be thorough.

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx [hunk 2]
```diff
@@ -21,41 +21,56 @@ const GiftAMealPage = (props: Props) => {
...
-    // Inactive campaigns should show up newest to oldest.
-    setPastCampaigns(past.reverse());
   };
 
   useEffect(() => {
     fetchData();
     // eslint-disable-next-line
   }, []);
 
+  useEffect(() => {
```
Comment (jacoblb64) [inline src/components/MerchantsPage/gam/GiftAMealPage.tsx:56]: What's the reason for adding another `useEffect()` call? Can we combine these?

**Review by jacoblb64 — APPROVED** [review]
Looks good to me! Cleaner and less calls which is always good :) Left one comment/question but non-blocking.

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx (hunk 2 above)
Comment (wilsonj806) [reply src/components/MerchantsPage/gam/GiftAMealPage.tsx:56]: Don't think so, cuz the way I have the effect for fetching past campaigns set up is that it needs some state in order to synchronize, whereas the first one doesn't really depend on anything to synchronize.

**Pull request merged**